app/jobs/clean_stale_uphold_data_job.rb

```diff
@@ -0,0 +1,34 @@
+class CleanStaleUpholdDataJob < ApplicationJob
+  queue_as :scheduler
+
+  def perform
+    require "sentry-raven"
+    # clear uphold codes sitting for over 5 minutes   
+    publishers = Publisher.has_stale_uphold_code
+    n = 0
+    publishers.each do |publisher|
+      raise if publisher.uphold_status != :code_acquired
+      publisher.uphold_code = nil
+      publisher.save!
+      n += 1
+      Rails.logger.info("Cleaned stalled uphold code for #{publisher.id}.")
+      Raven.capture_message("Cleaned stalled uphold code for #{publisher.id}.")
+    end
+    Rails.logger.info("CleanStaleUpholdDataJob cleared #{n} stalled uphold codes.")
+    Raven.capture_message("CleanStaleUpholdDataJob cleared #{n} stalled uphold codes.")
```

Comment:
the correct way to use capture_message is to do
```rb
Raven.capture_message("A nonchanging message", param_one: "apple")
```

having the messages share the same content is necessary to make sentry work nicely

Reply:
Noted, thanks.